alcove/query/api.py

```
@@ -19,8 +29,136 @@
+
+            if request.url.path == "/view" and 200 <= status < 300:
+                body_parts = []
+                async for chunk in response.body_iterator:
+                    if isinstance(chunk, bytes):
+                        body_parts.append(chunk)
+                    else:
+                        body_parts.append(chunk.encode("utf-8"))
+                body_bytes = b"".join(body_parts)
+                doc_chars = len(body_bytes)
+                if doc_chars > _largest_view_chars:
+                    _largest_view_chars = doc_chars
```

Ruff (0.15.5)

[warning] Replace `if` statement with `max` call

Replace with `max` call

(PLR1730)